.gitignore

```diff
@@ -83,6 +83,7 @@ man/*.1
 man/*.3
 man/*.8
 man/.doctrees/
+nbproject/
```

Comment:
I know we have *~ but that is erroneous and unnecessary and we really should remove it.

Editors are varied and leave all kinds of trash around.  We don't need all of the different editor files in .gitignore. You should add it to `~/.config/git/ignore` or wherever you have your global `core.excludesfile` set, as documented at https://git-scm.com/docs/git-config

Reply:
Changes submitted. Please check the newest version og .gitignore